### PR TITLE
Remove vc-js library

### PR DESCRIPTION
**Pull request by SamHellawell**
Resolves:
https://github.com/docknetwork/sdk/issues/180
https://github.com/docknetwork/sdk/issues/194

Side effects of this PR: I added linting for tests and ran fix on them, they arent required to pass for CI but i think we should keep them cleaned too

### CODE REVIEW

File: src/utils/vc/credentials.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+*/
+
+/**
+ * Verify a Verifiable Credential. Returns the verification status and error in an object
+ * @param {object} [credential] The VCDM Credential
+ * @param {VerifiableParams} options Verify parameters, this object is passed down to jsonld-signatures calls
+ * @return {Promise<object>} verification result. The returned object will have a key `verified` which is true if the
+ * credential is valid and not revoked and false otherwise. The `error` will describe the error if any.
+ */
+// TODO: perform verification on the expanded JSON-LD credential
```
Comment (SamHellawell) [inline src/utils/vc/credentials.js:155]: i have moved this TODO into another issue as itll effect too much logic for this PR, imo. better to address separately

**Comment by SamHellawell** [comment]
I have opened this issue https://github.com/docknetwork/sdk/issues/196 to address incorrect de-referencing of properties, itll require a bit of logic reworking and maybe some test/example adjustment so will address in future PR

File: src/utils/revocation.js [hunk 2]
```diff
@@ -17,8 +38,97 @@ export function createRandomRegistryId() {
   return randomAsHex(RevRegIdByteSize);
 }
 
+/**
```
Comment (SamHellawell) [inline src/utils/revocation.js:41]: i moved a few methods from utils/vc into this file as they can all be considered as vc revocation

File: src/utils/vc/CredentialIssuancePurpose.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+/*!
```
Comment (SamHellawell) [inline src/utils/vc/CredentialIssuancePurpose.js:1]: taken from vc-js library, so left copyright in

File: tests/unit/claimgraph.test.js [hunk 4]
```diff
@@ -2,29 +2,29 @@ import { fromJsonldjsCg, asEE, merge } from '../../src/utils/claimgraph';
 
 describe('Claimgraph operations.', () => {
   test('merge', async () => {
-    let cg1 = [
```
Comment (SamHellawell) [inline tests/unit/claimgraph.test.js:5]: changes to this file are from linting

File: tests/unit/canonicalize.test.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-import { canon } from '../../src/utils/canonicalize';
 import deepEqual from 'deep-equal';
+import { canon } from '../../src/utils/canonicalize';
```
Comment (SamHellawell) [inline tests/unit/canonicalize.test.js:2]: changes to this file are from linting

**Review by lovesh — APPROVED** [review]

**Pull request merged**